src/provider/provider_tracking.c

```diff
@@ -371,7 +371,7 @@ static umf_result_t trackingFree(void *hProvider, void *ptr, size_t size) {
 
     ret = umfMemoryProviderFree(p->hUpstream, ptr, size);
     if (ret != UMF_RESULT_SUCCESS) {
-        LOG_ERR("upstream provider is failed to free the memory");
+        LOG_ERR("upstream provider failed to free the memory");
```

Comment:
I believe I've seen this exact phrasing in more than one place, pls fix all of them

Reply:
@lukaszstolarczuk Done